sparse_ho/models/svr.py

```diff
@@ -392,17 +354,8 @@ def generalized_supp(self, X, v, log_hyperparam):
         return v[full_supp]
 
     def proj_hyperparam(self, X, y, log_hyperparam):
-        if log_hyperparam[0] < -16.0:
-            log_hyperparam[0] = -16.0
-        elif log_hyperparam[0] > 2:
-            log_hyperparam[0] = 5
-
-        if log_hyperparam[1] < -16.0:
-            log_hyperparam[1] = -16.0
-        elif log_hyperparam[1] > 2:
-            log_hyperparam[1] = 2
-
-        return log_hyperparam
+        return np.array([np.clip(log_hyperparam[0], -16, 5),
```

Comment:
I think you can do np.clip(log_hyperparam, -16, [5, 2])

Reply:
Thanks I'll try that